src/lib/sections/Welcome.svelte

```diff
@@ -3,27 +3,27 @@
   id="welcome"
 >
   <div
-    class="dark:bg-welcome-dark absolute left-0 top-0 -z-10 h-full w-full bg-welcome-light bg-[length:cover,auto] bg-center bg-no-repeat"
+    class="absolute left-0 top-0 -z-10 h-full w-full bg-welcome-dark bg-[length:cover,auto] bg-center bg-no-repeat dark:bg-welcome-dark"
   />
   <div class="relative mx-auto w-11/12 md:w-10/12">
     <div
-      class="absolute bottom-5 left-3 h-full w-full bg-gradient-to-br from-[#C5083A] to-[#FF99B4] dark:border-2 dark:border-white md:left-5"
+      class="absolute bottom-5 left-3 h-full w-full border-2 border-white bg-gradient-to-br from-[#e67a88] to-[#f2d3d7] dark:border-white dark:from-[#00000000] dark:to-[#00000000] md:left-5"
```

Comment:
The hex codes are invalid, but more importantly, a gradient isn't necessary if it's going from one color to another color.

Reply:
that's the hex code for transparency. I wasn't sure how to indicate not have a gradient for dark. Like how do i override bg-gradient-to-br when i do "dark:"